xtest-blas/tests/oper.rs

```diff
@@ -52,6 +55,20 @@ fn range_mat64(m: Ix, n: Ix) -> Array2<f64> {
         .unwrap()
 }
 
+fn range_mat_complex(m: Ix, n: Ix) -> Array2<Complex32> {
+    Array::linspace(0., (m * n) as f32 - 1., m * n)
```

Comment:
Would it be possible to test with not purely real matrices?

Reply:
Added a test that exercises both complex entries in the matrix and also complex alpha/beta.